testsuite/cluster-test/src/experiments/load_test.rs

```diff
@@ -124,15 +125,21 @@ impl Experiment for LoadTest {
         // await on all spawned tasks
         tokio::time::delay_for(Duration::from_secs(self.duration)).await;
         if let Some(j) = emit_job {
-            let _ = context.tx_emitter.stop_job(j).await;
+            let stats = context.tx_emitter.stop_job(j).await;
+            info!("Total tx emitter stats: {}", stats);
```

Comment:
does this include actual bytes of txns submitted to JSON-RPC? would be nice to add if not

Reply:
our tx is using same script as the payload, so we do expect each tx has same bytes. Since tx_emitter stat is widely used currently. I added a dummy tx to calculate how is the actual number of bytes submitted to JSON-RPC here instead of making change on tx_emitter stats